README.md

```diff
@@ -1 +1,49 @@
-# MicroverseStudent
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello-Microverse
+
+> "Hello Microverse" project
+
+
+## Built With
+
+- HTML/CSS
+
+
+## Getting Started
+
+**This is the first repository created for the Microverse**
+
+This file was made for review.
```

Comment:
[OPTIONAL]

Let's say for this Hello World project it's acceptable (but not recommended) to not have all the sections of the README file completed, but from now on, on each project, we always have to complete all the README sections.